src/status_im2/common/bottom_sheet/view.cljs

```diff
@@ -66,22 +74,24 @@
                 {:opacity bg-opacity}
                 {:flex 1 :background-color colors/neutral-100-opa-70})}]]
      ;; sheet
-     [gesture/gesture-detector {:gesture sheet-gesture}
-      [reanimated/view
-       {:style (reanimated/apply-animations-to-style
-                {:transform [{:translateY translate-y}]}
-                (styles/sheet insets window-height override-theme shell?))}
+     (cond->>
+       [reanimated/view
+        {:style (reanimated/apply-animations-to-style
+                 {:transform [{:translateY translate-y}]}
+                 (styles/sheet insets window-height override-theme padding-bottom-override shell?))}
 
-       (when shell?
-         [blur/view
-          {:style styles/shell-bg}])
+        (when shell?
+          [blur/view
+           {:style styles/shell-bg}])
 
-       (when selected-item
-         [rn/view
-          [rn/view {:style (styles/selected-item override-theme)}
-           [selected-item]]])
+        (when selected-item
+          [rn/view
+           [rn/view {:style (styles/selected-item override-theme)}
+            [selected-item]]])
 
-       ;; handle
-       [rn/view {:style (styles/handle override-theme)}]
-       ;; content
-       [content]]]]))
+        ;; handle
+        [rn/view {:style (styles/handle override-theme)}]
+        ;; content
+        [content]]
+       enable-scroll?
```

Comment:
could you elaborate pls

Reply:
This is unneeded anymore, I added it before to enable scrolling inside bottom sheet by disabling gesture handler